scripts/clean_protos.sh

```diff
@@ -0,0 +1,40 @@
+#!/bin/bash
+set -o errexit -o nounset -o pipefail -x
+command -v shellcheck > /dev/null && shellcheck "$0"
+
+
+# Usage: clean_protos.sh [outdir]
+# It will look for all protobuf files, and store them in 
+# the same tree structure in an output directory (default spec)
+# for easy import by other projects.
+#
+# We produce two versions
+# - spec/gogo verbatim copy for use by other go repos
+# - spec/cleaned cleaned up by cleanproto for use in other languages.
+
+
+(cd cmd/cleanproto && make build)
+CLEAN=./cmd/cleanproto/cleanproto
+
+# OUT_DIR=${1:-spec}
+OUT_DIR=spec
+rm -rf ${OUT_DIR}
+
+(
+  find . -name '*.proto' -not -path '*/vendor/*' -not -path '*/examples/*' -not -path '*/cmd/bcpd/*' -not -path spec > tmp
```

Comment:
You can directly pipe `find` result to `read` (iterate over the results). This way you don't need to create `tmp` file and remove it later.